packages/@azure/servicebus/data-plane/test/testUtils.ts

```diff
@@ -193,19 +192,26 @@ export function getReceiverClient(
 }
 
 export async function purge(
-  recieverClient: QueueClient | SubscriptionClient,
+  receiverClient: QueueClient | SubscriptionClient,
   useSessions?: boolean
 ): Promise<void> {
-  const peekedMsgs = await recieverClient.peek();
-  if (!peekedMsgs.length) {
-    return;
-  }
+  let isEmpty = false;
 
   const receiver = useSessions
-    ? await recieverClient.getSessionReceiver()
-    : recieverClient.getReceiver();
+    ? await receiverClient.getSessionReceiver({ sessionId: "my-session" })
```

Comment:
Any reason to add the session id here?
I had previously omitted it to cover the case where you could be using a queue which has messages with a different session id?